e2e2/test/cases/neuron/manifests/bert-train.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: bert-mpi-training
+spec:
+  slotsPerWorker: 1
+  runPolicy:
+    backoffLimit: 20
+    cleanPodPolicy: Running
+  mpiReplicaSpecs:
+    Launcher:
+      replicas: 1
+      template:
+        spec:
+          restartPolicy: OnFailure
+          containers:
+          - image: public.ecr.aws/o5d5x8n6/weicongw:neuron
+            imagePullPolicy: Always
+            name: bert-training-launcher
+            env:
+            - name: NEURON_CORES
+              value: "32"
```

Comment:
This value should also reflect the change in the number of replicas. Once the number of replicas is modified to 4, then the number here should be 64. Or if parameterized, 16 * {{# of replicas}}

Reply:
Sorry for the confusion, it means the neuron core of each node. Updated the env name to `NEURON_CORES_PER_NODE`

Reply:
Wouldn't the number of neuron cores be 16 for each `trn1.32xlarge` instance?

Reply:
It has 32 neuron cores but 16 neuron devices.
```
  aws.amazon.com/neuron:        16
  aws.amazon.com/neuroncore:    32
  aws.amazon.com/neurondevice:  16
```

Reply:
Thanks. Finally found the docs explaining 2 neuroncore per neurondevice.